### PR TITLE
Login view

### LINKED ISSUE #4: Admin login
# REQUIREMENTS
Layout of the Admin Login (Form to login system admin)

# DESIGN
The layout of this view should look like this mockup

image

### PR DESCRIPTION
**Pull request by dany-herz42**
# Description
We create the Login view and controller. also we fixed some errors with the styles for the others views.

## Screenshots
image

### CODE REVIEW

File: checkClock/app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import './scss/clearStyles.scss';
+@import './scss/variables.scss';
+
+@import './scss/home.scss';
+@import './scss/AdminDashboard.scss';
+// @import './scss/login.scss';
```
Comment (JuanequeX) [inline checkClock/app/assets/stylesheets/main.scss]: Add a blank line here and if don't need this import commented should be deleted it

File: checkClock/app/assets/stylesheets/scss/login.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        border: none;
+        color: $font-color-primary;
+        font-weight: 700;
+        padding-bottom: 10px;
+        padding-top: 10px;
+        width: 40%;
+        font-size: 1.2em;
+        letter-spacing: 2px;
+        cursor: pointer;
+        margin-top: 15px;
```
Comment (JuanequeX) [inline checkClock/app/assets/stylesheets/scss/login.scss]: Sort this lines

File: checkClock/app/assets/stylesheets/scss/login.scss [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      input {
+        width: 80%;
+        padding: 10px 0px;
+        text-align: center;
+        border: 1px solid $color-secondary-light;
+        border-radius: 6px;
+        background-color: transparent;
+        font-size: 1.2em;
+        color: $font-color-secondary;
+
```
Comment (JuanequeX) [inline checkClock/app/assets/stylesheets/scss/login.scss]: Same with these block, please sort it

File: checkClock/app/assets/stylesheets/scss/login.scss [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        border: 1px solid $color-secondary-light;
+        border-radius: 6px;
+        background-color: transparent;
+        font-size: 1.2em;
+        color: $font-color-secondary;
+        
+      }
+    }
+  }
+}
```
Comment (JuanequeX) [inline checkClock/app/assets/stylesheets/scss/login.scss]: Add a blank line here

File: checkClock/test/controllers/login_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class LoginControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (JuanequeX) [inline checkClock/test/controllers/login_controller_test.rb]: If this code are not needed please delete it

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: checkClock/app/helpers/login_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module LoginHelper
+end
```
Comment (YilKanda) [inline checkClock/app/helpers/login_helper.rb]: If you don't occupy, remove it.

File: checkClock/app/views/home/index.html.erb [hunk 7]
```diff
@@ -1,61 +1,51 @@
...
-    </div>
-  </nav>
-  <main>
-    <div class="col-left">
-      <div class="date">
-        <p id="date">o</p>
-      </div>
+<nav class="home-nav">
+  <div class="banner">
+    <div class="logo">
```
Comment (YilKanda) [inline checkClock/app/views/home/index.html.erb:3]: Can you put your banner in an external file?

File: checkClock/app/views/login/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+        <label for="password">Password: </label>
+        <input type="password" name="password" id="password" placeholder="Type your password here">
+      </div>
+      <div class="form-row">
+        <input type="submit" value="Log In">
+      </div>
+    </form>
+  </div>
+</main>
+<%= javascript_pack_tag 'analogic' %>
```
Comment (YilKanda) [inline checkClock/app/views/login/index.html.erb]: Add a blank line.

File: checkClock/app/views/home/index.html.erb (hunk 7 above)
Comment (dany-herz42) [reply checkClock/app/views/home/index.html.erb:3]: Yes, we will implement this recommendation in future commits

File: checkClock/app/views/login/index.html.erb (hunk 8 above)
Comment (dany-herz42) [reply checkClock/app/views/login/index.html.erb]: Done

File: checkClock/app/helpers/login_helper.rb (hunk 6 above)
Comment (dany-herz42) [reply checkClock/app/helpers/login_helper.rb]: Done

File: checkClock/test/controllers/login_controller_test.rb (hunk 5 above)
Comment (dany-herz42) [reply checkClock/test/controllers/login_controller_test.rb]: Done

File: checkClock/app/assets/stylesheets/scss/login.scss (hunk 4 above)
Comment (dany-herz42) [reply checkClock/app/assets/stylesheets/scss/login.scss]: Done

File: checkClock/app/assets/stylesheets/scss/login.scss (hunk 3 above)
Comment (dany-herz42) [reply checkClock/app/assets/stylesheets/scss/login.scss]: Done

File: checkClock/app/assets/stylesheets/scss/login.scss (hunk 2 above)
Comment (dany-herz42) [reply checkClock/app/assets/stylesheets/scss/login.scss]: Done

File: checkClock/app/assets/stylesheets/main.scss (hunk 1 above)
Comment (dany-herz42) [reply checkClock/app/assets/stylesheets/main.scss]: Done

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**